### PR TITLE
Make the changes for the icon

### PR DESCRIPTION
**Pull request by Nagol12344**
This fixes a bug with the icon on linux.

### CODE REVIEW

File: src/electron.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ function createWindow() {
       contextIsolation: false,
       enableRemoteModule: false,
     },
+    icon: path.resolve(app.getAppPath(), 'src', 'uc.png'),
```
Comment (feildmaster) [inline src/electron.js:14]: Is it not possible to use the existing icon location?

File: package.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
     "electron-builder": "^22.9.1"
   },
   "dependencies": {
+    "@types/fs-extra": "^9.0.13",
```
Comment (feildmaster) [inline package.json]: This should be a dev-dependency

File: src/electron.js (hunk 1 above)
Comment (Nagol12344) [reply src/electron.js:14]: Where is it, I never saw that

Comment (feildmaster) [reply src/electron.js:14]: `build/icon.png`

File: scripts/changelog/index.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /* eslint-env node */
-const fs = require('fs');
+const fs = require('fs').promises;
```
Comment (feildmaster) [inline scripts/changelog/index.js]: Is this necessary? Pretty sure the existing code doesn't use a promise here

Comment (Nagol12344) [reply scripts/changelog/index.js]: i was thinking that may solve a error i was having.

File: src/electron.js (hunk 1 above)
Comment (Nagol12344) [reply src/electron.js:14]: that seems to make it not load.

File: package.json (hunk 2 above)
Comment (Nagol12344) [reply package.json]: Fixed!

File: scripts/changelog/index.js (hunk 3 above)
Comment (Nagol12344) [reply scripts/changelog/index.js]: Fixed!

File: src/electron.js (hunk 1 above)
Comment (feildmaster) [reply src/electron.js:14]: Got it

**Review by feildmaster — APPROVED** [review]
Looks good.

**Pull request merged**